packages/create-cloudflare/tsconfig.json

```diff
@@ -1,8 +1,15 @@
 {
 	"root": true,
 	"extends": "@cloudflare/workers-tsconfig/tsconfig.json",
-	"include": ["**/*.ts"],
-	"exclude": ["node_modules", "dist", "templates", "**/templates/**"],
+	"include": ["**/*.ts", "templates/*/c3.ts"],
```

Comment:
I think the second item here is already covered by the first, no?

Reply:
This is trying to ensure that we exclude type checking on all files in `templates/` with the exception of the `c3.ts` files which live in the top level of each template directory.

I think because I added `"templates/*/*/**"` to exclude, I need to add `"templates/*/c3.ts"` because of specificity issues to explicitly add back the `c3.ts` file. This took a lot of head scratching, if there's a better way of doing this please lmk 😄